### ISSUE TITLE
When automatically wheel, pageController always stays in the final position

### ISSUE THREAD

**Issue by superzcj** [DeveloperPans/PSCarouselView#7]
When automatically wheel, pageController always stays in the final position.

**Comment by MrPans** [comment]
implement delegate and set currentPage:

```

- (void)carousel:(PSCarouselView *)carousel didMoveToPage:(NSUInteger)page
{
    NSLog(@"PSCarouselView did MOVE to No.%ld page",page);
    self.pageControl.currentPage = page;
}

```

*_Make sure you had set `carouseView.pageDelegate` *_

```

    self.carouselView.pageDelegate = self;

```

**Comment by superzcj** [comment]
Yes,I have done in my program.
In demo,the bug is still.The pageController stays red position.
And manually wheel,program is normal.

**Comment by MrPans** [comment]
**thx for your bug report!**
this bug has been fixed in latest version(1.0.5).
**use `pod update` to fix this problem.**
